libcodechecker/libhandlers/store.py

```diff
@@ -238,11 +238,19 @@ def consume_plist(item):
 
         host, port = server_data
         client = setup_client(host, port, '/')
-        rh.handle_results(client)
-        return 0
+        return rh.handle_results(client)
     except Exception as ex:
         LOG.warning("Failed to process report: " + f)
-        return 1
+        return []
+
+
+def consumed_results(client, run_id):
+    def markFixed(l):
```

Comment:
(Let's keep not introducing style warnings and keep to the Pythonic nomenclature, `mark_fixed`. Also comment below is not a full sentence. I'm Vice Xazax now! 😛 )